src/components/GGanttChart.vue

```diff
@@ -68,6 +81,9 @@ export interface GGanttChartProps {
   rowHeight?: number
   highlightedUnits?: number[]
   font?: string
+  columnTitle?: string
```

Comment:
A more specific name would be suitable, such as `labelColumnTitle`and `labelColumnWidth`. Also, it is excellent that this prop is optional! Users have a choice, then!

Reply:
adjusted